azurelinuxagent/common/utils/fileutil.py

```diff
@@ -123,6 +124,15 @@ def chmod(path, mode):
         os.chmod(path, mode)
 
 
+def umount(*args):
+    for paths in args:
+        # find all possible mounts
+            for path in glob.glob(paths):
+                if os.path.ismount(path):
+                    command=["umount",path]
+                    shellutil.run_command(command)
```

Comment:
Is the usage of the `command` var necessary here? Looks like a single-use variable, consider combining this into a single line.

Reply:
I agree, we can shorten this to one line.